public/locales/sp/translation.json

```diff
@@ -816,7 +816,7 @@
     "title": "Configuración",
     "general": "General",
     "actionItemCategories": "Categorías de elementos de acción",
-    "updateOrganization": "Actualizar organización",
+    "Edit Organization": "Editar organización",
```

Comment:
This is not in keeping with the camel case standard. Remove the space, use camel case and update the source code.